docs/modules/ROOT/pages/spring-cloud-netflix.adoc

```diff
@@ -131,6 +131,17 @@ public class RestClientConfiguration {
 }
 ----
 
+If you want to control Apache HttpClient 5's protocol specifically, then you can achieve the same
+with the property `eureka.client.http-components.enable-protocol-upgrades`. For example:
```

Comment:
Please add info that property settings will be overridden by user-provided `EurekaClientHttpRequestFactorySupplier.RequestConfigCustomizer` bean.